lib/config.js

```diff
@@ -176,7 +176,8 @@ Config.prototype.buildProjects = function () {
       url: getNPMConfig(['projects', projectName, 'repository', 'url']),
       gclientName: getNPMConfig(['projects', projectName, 'dir']),
       dir: path.join(this.rootDir, getNPMConfig(['projects', projectName, 'dir'])),
-      custom_deps: packages.config.projects[projectName].custom_deps
+      custom_deps: packages.config.projects[projectName].custom_deps,
+      arg_name: projectName.replace('-', '_')
```

Comment:
maybe just `name`? `arg_` was a little confusing to me. Just a nit though.

Reply:
Didn't want it to get confused with the actual name since the original name is still being used in other places